mule/meta.yaml

```diff
@@ -0,0 +1,35 @@
+package:
+    name: mule
+    version: 1.2
+
+source:
+    fn: mule.tar.gz
+    url: https://github.com/SciTools/mule/releases/download/v1.2/mule.tar.gz
```

Comment:
@stevewardle: This should be https://github.com/SciTools/mule, the version number you have added at the top there will tell conda-build to pick up the right version.